src/vm/stubgen.h

```diff
@@ -503,13 +502,24 @@ class ILStubLinker
     void SetStubTargetReturnType(CorElementType typ);
     void SetStubTargetReturnType(LocalDesc* pLoc);
     void SetStubTargetCallingConv(CorCallingConvention uNativeCallingConv);
+    
+    int GetReturnStackDelta()
+    {
+        if ((!m_fIsReverseStub && m_StubHasVoidReturnType) || (m_fIsReverseStub && m_StubTargetHasVoidReturnType))
```

Comment:
This function only returns 1 of 2 values. Can we return a boolean or rework how we get the information?